rsmtool/analyzer.py

```diff
@@ -590,15 +596,37 @@ def metrics_helper(human_scores,
         system_scores: pandas Series
             Series containing numeric scores predicted by the model.
         population_human_score_sd : float, optional
-            Reference standard deviation for human scores. This is used to compute SMD and
-            should be the standard deviation for the whole population when SMD are computed
-            for individual subgroups.
-            When None, this will be computed as the standard deviation of `human_scores`
+            Reference standard deviation for human scores. If `smd_method='williamson'`, this is
+            used to compute SMD and should be the standard deviation for the whole population. If
+            `use_std_diff=True`, this must be used with `population_human_score_mn`.
+            Otherwise, it is ignored.
+            Defaults to None.
         population_system_score_sd : float, optional
-            Reference standard deviation for system scores.  This is used to compute SMD and
-            should be the standard deviation for the whole population when SMD are computed
-            for individual subgroups.
-            When None, this will be computed as the standard deviation of `system_scores`.
+            Reference standard deviation for system scores. If `smd_method='williamson'`, this is
+            used to compute SMD and should be the standard deviation for the whole population.If
+            `use_std_diff=True`, this must be used with `population_human_score_mn`.
+            Otherwise, it is ignored.
+            Defaults to None.
+        population_human_score_mn : float, optional
+            Reference mean for human scores. If `use_std_diff=True`, this must be used with
+            `population_human_score_mn`. Otherwise, it is ignored.
+            Defaults to None.
+        population_system_score_mn : float, optional
+            Reference mean for system scores. If  `use_std_diff=True`, this must be used with
+            `population_human_score_mn`. Otherwise, it is ignored.
+            Defaults to None.
+        smd_method : {'williamson', 'johnson', pooled', 'unpooled'}, optional
```

Comment:
I would like to re-think this method going forward is it is a bit confusing right now. Let's wait till we have more clarity about various SMD and DSM though.